### PR TITLE
docs(blog): add electron 25.0.0 blog

### PR DESCRIPTION
**Pull request by BlackHole1**

### CODE REVIEW

**Review by sriting — DISMISSED** [review]

File: blog/electron-25-0.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  slug: electron-25-0
+---
+
+Electron 25.0.0 已发布！此次升级中包含 Chromium `114`，V8 `11.4`，和 Node.js `18.15.0`。请阅读下文了解更多详情！
+
+---
+
+Electron 团队很高兴发布了 Electron 25.0.0 ！您可以通过 `npm install electron@latest` 进行安装，或者从我们的 [发布网站](https://releases.electronjs.org/releases/stable) 下载它。 继续阅读此版本的详细信息。
+
+如果您有任何反馈，请在Twitter上与我们分享，或加入我们的社区 [Discord](https://discord.com/invite/electronjs)！Bug 和功能请求可以在 Electron 的 [问题跟踪器](https://github.com/electron/electron/issues) 中报告。
```
Comment (LitoMore) [inline blog/electron-25-0.md]:
```suggestion
如果您有任何反馈，请在 Twitter 上与我们分享，或加入我们的社区 [Discord](https://discord.com/invite/electronjs)！Bug 和功能请求可以在 Electron 的 [问题跟踪器](https://github.com/electron/electron/issues) 中报告。
```

File: blog/electron-25-0.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+});
+```
+
+```js
+// 在 Electron 25 中弃用
+protocol.registerFileProtocol('some-protocol', () => {
+  callback({ filePath: '/path/to/my/file' });
+});
+
+// 代替为
```
Comment (LitoMore) [inline blog/electron-25-0.md]:
```suggestion
// 替换为
```

File: blog/electron-25-0.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+});
+```
+
+```js
+// 在 Electron 25 中弃用
+protocol.registerHttpProtocol('some-protocol', () => {
+  callback({ url: 'https://electronjs.org' });
+});
+
+// 代替为
```
Comment (LitoMore) [inline blog/electron-25-0.md]:
```suggestion
// 替换为
```

File: blog/electron-25-0.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+- 向 `webContents` 和 `webview` 标签添加了一个 `will-frame-navigate` 事件，每当 frame 层中的任何一个 frame 调用 navigate 时，都会触发该函数。[#34418](https://github.com/electron/electron/pull/34418)
+- 向 navigator 事件添加了启动信息。这个信息允许区分来自父框架 `window.open` 引起的 navigation 事件，而不是来自子框架发起的 navigation 事件。[#37085](https://github.com/electron/electron/pull/37085)
+- 增加了 net.resolveHost，它使用 defaultSession 对象来解析 hosts。[#38152](https://github.com/electron/electron/pull/38152)
+- 向 `app` 添加了新的 'did-resign-active' 事件。[#38018](https://github.com/electron/electron/pull/38018)
+- 向 `webContents.print()` 添加了几个标准页面大小选项。[#37159](https://github.com/electron/electron/pull/37159)
+- 向会话处理程序 `ses.setDisplayMediaRequestHandler()` 的回调中添加了 `enableLocalEcho` 标志，以允许当 `audio` 为 `WebFrameMain` 时在本地输出流中回显远程音频输入。[#37315](https://github.com/electron/electron/pull/37315)
+- 向 `powerMonitor` 添加了 thermal 信息管理。[#38028](https://github.com/electron/electron/pull/38028)
+- 允许将绝对路径传递给 session.fromPath() API。[#37604](https://github.com/electron/electron/pull/37604)
+- 在 `webContents` 上公开 `audio-state-changed` 事件。[#37366](https://github.com/electron/electron/pull/37366)
```
Comment (LitoMore) [inline blog/electron-25-0.md]:
```suggestion
- 在 `webContents` 上暴露出了 `audio-state-changed` 事件。[#37366](https://github.com/electron/electron/pull/37366)
```

File: blog/electron-25-0.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+### 新特性
+
+- 已添加 `net.fetch()`。[#36733](https://github.com/electron/electron/pull/36733)
+  - `net.fetch` 支持对 `file:` URL 和使用 `protocol.register*Protocol` 注册的自定义协议的请求。
+- 添加了 BrowserWindow.set/getWindowButtonPosition API。[#37094](https://github.com/electron/electron/pull/37094)
+- 添加了 `protocol.handle`，替换并弃用了 `protocol.{register,intercept}{String,Buffer,Stream,Http,File}Protocol`。[#34418](https://github.com/electron/electron/pull/34418)
+- 向 `webContents` 和 `webview` 标签添加了一个 `will-frame-navigate` 事件，每当 frame 层中的任何一个 frame 调用 navigate 时，都会触发该函数。[#34418](https://github.com/electron/electron/pull/34418)
+- 向 navigator 事件添加了启动信息。这个信息允许区分来自父框架 `window.open` 引起的 navigation 事件，而不是来自子框架发起的 navigation 事件。[#37085](https://github.com/electron/electron/pull/37085)
+- 增加了 net.resolveHost，它使用 defaultSession 对象来解析 hosts。[#38152](https://github.com/electron/electron/pull/38152)
+- 向 `app` 添加了新的 'did-resign-active' 事件。[#38018](https://github.com/electron/electron/pull/38018)
```
Comment (LitoMore) [inline blog/electron-25-0.md]:
```suggestion
- 向 `app` 添加了新的 `did-resign-active` 事件。[#38018](https://github.com/electron/electron/pull/38018)
```

**Review by LitoMore — CHANGES_REQUESTED** [review]

**Comment by BlackHole1** [comment]
@LitoMore Done

**Review by LitoMore — APPROVED** [review]

**Pull request merged**